code/player/controller/DefaultWalkController.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+
+using Sandbox;
+
+namespace TTTReborn.Player
+{
+    public partial class DefaultWalkController : WalkController
+    {
+        [ServerVar]
+        public static bool IsSprintEnabled { get; set; } = false;
+
+        public float MaxSprintSpeed = 300f;
+        public float StaminaLossPerSecond = 30f;
+        public float StaminaGainPerSecond = 25f;
+
+        private bool _jumped = false;
```

Comment:
Is this used anywhere?

Reply:
ty for the hint, used in previous testing and I forget to remove it ^^